docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -112,9 +112,8 @@ If the value is an array: multiple URL templates. Each endpoint must return the
 
 If supplied, `getTileData` is called to retrieve the data of each tile. It receives one argument `tile` which contains the following fields:
 
-- `x` (Number) - x index of the tile
-- `y` (Number) - y index of the tile
-- `z` (Number) - z index of the tile
+- `index` (Object) - index of the tile. `index` is in the shape of `{x, y, z}`, corresponding to the integer values specifying the tile.
+- `cacheKey` (String) - unique string representation of index, as 'x-y-z', e.g. '0-2-3'.
```

Comment:
The rest of the code base would just name this kind of property `id`

Reply:
I've renamed to `id`